### PR TITLE
code beautify

### PR DESCRIPTION
**Pull request by muhammedfurkan**

### CODE REVIEW

File: aiogram/types/inline_keyboard.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def add(self, *args):
             if index % self.row_width == 0:
                 self.inline_keyboard.append(row)
                 row = []
-        if len(row) > 0:
+        if row:
```
Comment (evgfilim1) [inline aiogram/types/inline_keyboard.py:53]: Explicit is better than implicit

File: aiogram/types/reply_keyboard.py [hunk 2]
```diff
@@ -58,7 +58,7 @@ def add(self, *args):
             if index % self.row_width == 0:
                 self.keyboard.append(row)
                 row = []
-        if len(row) > 0:
+        if row:
```
Comment (evgfilim1) [inline aiogram/types/reply_keyboard.py:61]: Explicit is better than implicit

File: tests/test_utils/test_deprecated.py [hunk 3]
```diff
@@ -7,8 +7,8 @@ def test_DeprecatedReadOnlyClassVarCD():
     assert DeprecatedReadOnlyClassVar.__slots__ == ("_new_value_getter", "_warning_message")
 
     new_value_of_deprecated_cls_cd = "mpa"
-    pseudo_owner_cls = type("OpekaCla$$", (), {})
```
Comment (evgfilim1) [inline tests/test_utils/test_deprecated.py:10]: Why have you moved it?

File: aiogram/types/inline_keyboard.py (hunk 1 above)
Comment (uburuntu) [reply aiogram/types/inline_keyboard.py:53]: https://google.github.io/styleguide/pyguide.html#2144-decision

File: aiogram/types/fields.py [hunk 4]
```diff
@@ -126,42 +126,32 @@ def __init__(self, *args, **kwargs):
         if default is None:
             default = []
 
-        super(ListField, self).__init__(*args, default=default, **kwargs)
+        super().__init__(*args, default=default, **kwargs)
```
Comment (Olegt0rr) [inline aiogram/types/fields.py:129]: I wanted to suggest, but it was applied 0_o

File: aiogram/utils/helper.py [hunk 5]
```diff
@@ -201,10 +194,15 @@ class OrderedHelperMeta(type):
...
+        props_keys = [
+            prop_name
+            for prop_name in (
+                name
+                for name, prop in namespace.items()
+                if isinstance(prop, (Item, ListItem))
+            )
+        ]
 
-        for prop_name in (name for name, prop in namespace.items() if isinstance(prop, (Item, ListItem))):
```
Comment (Olegt0rr) [inline aiogram/utils/helper.py:206]: Seems it's more readable

**Comment by JrooTJunior** [comment]
OMG :scream: 

That is... is.. is.. painful pull-request... I'm shocked...

Before start.. black, isort can't be applied to `dev-2.x` branch in due to this change breaks all opened pull-request and makes it impossible. I't breaks everything.

After partial reviewing you'r changes - tests was changed absolutely incorrect and code-style doesn't follows applied by our team rules.

For example:
```python
if <condition>:
    raise AssertionError
```
absolutelly incorrect in use instead of 
```python
assert <condition>
```
this change makes sense only when code will be started with `-O` interpreter option but not fully.

---

Also we don't planned to enable any automated code-checks for `dev-2.x` branch because already initialized development of `3.0` version where is enabled CI/CD with black, isort, flake8, mypy checks in strict mode. So if you want to make code prettified - lets look at `dev-3.x` branch.

---

I don't like to make negative or aggressive comments on issues/PR but... Changes should be applied with common sense and be adequate but you're changes are not adequate...

**Pull request closed without merging**